content/sensu-go/6.0/api/version.md

```diff
@@ -23,10 +23,10 @@ http://127.0.0.1:8080/version
 HTTP/1.1 200 OK
 {
   "etcd": {
-    "etcdserver": "3.3.17",
+    "etcdserver": "3.3.22",
     "etcdcluster": "3.3.0"
   },
-  "sensu_backend": "5.x.x#yyyyyyy"
+  "sensu_backend": "6.x.x#yyyyyyy"
```

Comment:
```suggestion
  "sensu_backend": "6.x.x"
```

Reply:
@nikkictl In the docs we say that the `yyyyyyy` thing is the SHA, not the date. Looks like it doesn't show up anymore either way though, is that right?

Reply:
The SHA does not appear in the `/version` API. It only appears in the `sensu-backend version`, `sensu-agent version` and `sensuctl version` commands.

```
$ sensuctl version
sensuctl version 6.0.0+ee, enterprise edition, build dd1adf3892a8f408c2abd3655fb8f15ea1d77bd0, built 2020-08-10T16:46:13Z, built with go1.13.15
```